tests/test_page.py

```diff
@@ -339,3 +341,70 @@ def test_is_minor_or_minor_version():
     assert page.version == "2.0"
     assert page.is_major_version() is True
     assert page.is_minor_version() is False
+
+
+@pytest.mark.parametrize(
+    "page_versions, expected_order",
+    (
+        (["1.0", "1.1", "1.2", "2.0"], ["2.0", "1.2", "1.1", "1.0"]),
+        (["2.0", "1.2", "1.1", "1.0"], ["2.0", "1.2", "1.1", "1.0"]),
```

Comment:
Might be worth having a more ranom order in the `page_versions` list too? Maybe a case something like
(["1.1","3.1", "1.2", "2.2", "1.0"], ["3.1", "2.2", "1.2", "1.1", "1.0"]),

Reply:
Ok, done.